### DISCUSSION TITLE
Visualize errors by OS

### DISCUSSION THREAD

**Discussion by saaaab1213** [getsentry/sentry-react-native#2658] [Q&A] [answered]
Hello everyone!

I am trying to find out errors related with only one OS. Is there a way to filter errors that happen only to iOS users?

**Comment by marandaneto** [comment, marked as answer]
Yes, https://docs.sentry.io/product/issues/ and https://docs.sentry.io/product/discover-queries/ have the ability to filter per many things, `os` being one of them.
